example/commitpr/main.go

```diff
@@ -125,7 +125,11 @@ func getFileContent(fileArg string) (targetName string, b []byte, err error) {
 // pushCommit creates the commit in the given reference using the given tree.
 func pushCommit(ref *github.Reference, tree *github.Tree) (err error) {
 	// Get the parent commit to attach the commit to.
-	parent, _, err := client.Repositories.GetCommit(ctx, *sourceOwner, *sourceRepo, *ref.Object.SHA)
+	opt := &github.ListOptions{
+		PerPage: 10,
+		Page:    1,
+	}
+	parent, _, err := client.Repositories.GetCommit(ctx, *sourceOwner, *sourceRepo, *ref.Object.SHA, opt)
```

Comment:
Since the focus of this example is not pagination, let's please just get rid of lines 128-131 and change line 132 to pass in `nil` instead of `opt`.